grails-app/services/com/recomdata/transmart/data/export/HighDimExportService.groovy

```diff
@@ -124,7 +124,19 @@ class HighDimExportService {
                         concept_key: term.key)
         ]
 
-        HighDimExporter exporter = highDimExporterRegistry.getExporterForFormat(format)
+        Set<HighDimExporter> exporters = highDimExporterRegistry.findExporters(fileFormat: format,
+                dataType: dataTypeResource.dataTypeName)
+
+        if (!exporters) {
+            throw new RuntimeException("No exporter was found for ${dataTypeResource.dataTypeName} data type" +
+                    " and ${format} file format.")
+        } else if (exporters.size() > 1) {
+            log.warn("There are more then one exporter for ${dataTypeResource.dataTypeName} data type" +
+                    " and ${format} file format. Using first one: ${exporters?.getAt(0)}")
+        }
```

Comment:
I made it support several exporters for a file format. I expect just one implementation for file format + data type pair though.